src/include/sof/lib/dai-zephyr.h

```diff
@@ -76,6 +76,16 @@ struct timestamp_ops {
 		      struct timestamp_data *tsd);
 };
 
+union hdalink_cfg {
+	uint16_t full;
+	struct {
+		uint16_t lchan	:4;
+		uint16_t hchan	:4;
+		uint16_t stream	:6;
+		uint16_t rsvd	:1;
+		uint16_t dir	:1;
+	} part;
+};
```

Comment:
Does this need to be packed or aligned ?

Reply:
I think it is not needed.